frontend/src/components/SessionEnd.js

```diff
@@ -0,0 +1,102 @@
+import SessionEndPlayerTile from './SessionEndPlayerTile';
+import styled from 'styled-components/macro';
+import ButtonTab from './ButtonTab';
+import ButtonBig from './ButtonBig';
+import { useState, useEffect } from 'react';
+
+export default function SessionEnd({ session, setFinalPlayerList }) {
+    const [playersWithScores, setPlayersWithScores] = useState([]);
+
+    useEffect(() => {
+        setPlayersWithScores(session.playerList);
+    }, []);
+
+    const onSubmitHandler = (event) => {
+        event.preventDefault();
+
+        const playerScores = playersWithScores.map((player) => player.score);
+
+        playerScores.sort(function (a, b) {
+            return b - a;
+        });
+
+        const winnerWithScore = playersWithScores.filter(
+            (player) => player.score === playerScores[0]
+        );
```

Comment:
What happens on a tie?

Why not sort the players with this compare function

```
function(playerA, playerB){
  return playerB.score - playerA.score;
}
```

Reply:
as for the MVP, I went with this version. thank for the pointer, though. in case of a tie I might have to introduce a winner state, which has to be changed by an additional event, if there are same scores for the winner... feels tricky ^^'